nupic/embodied/envs/vec_env.py

```diff
@@ -0,0 +1,244 @@
+"""
```

Comment:
It is missing the license header (should be in all files, please extend this comment to all files)